src/wallet/wallet.cpp

```diff
@@ -672,6 +673,59 @@ void CWallet::AddToSpends(const CWalletTx& wtx, WalletBatch* batch)
         AddToSpends(txin.prevout, wtx.GetHash(), batch);
 }
 
+void CWallet::UpdateAddressBookUsed(const CWalletTx& wtx)
+{
+    for (const auto& output : wtx.tx->vout) {
+        CTxDestination dest;
+        if (!ExtractDestination(output.scriptPubKey, dest)) continue;
+        m_address_book[dest].m_used = true;
+    }
+}
+
+void CWallet::InitialiseAddressBookUsed()
+{
+    for (const auto& entry : mapWallet) {
+        const CWalletTx& wtx = entry.second;
+        UpdateAddressBookUsed(wtx);
```

Comment:
2d3b9e24add7a9eb9824b60fc5590d3a1e4b8c6d can avoid a temporary in this loop and document with named arg instead

```diff
 void CWallet::InitialiseAddressBookUsed()
 {
     for (const auto& entry : mapWallet) {
-        const CWalletTx& wtx = entry.second;
-        UpdateAddressBookUsed(wtx);
+        UpdateAddressBookUsed(/*wtx=*/entry.second);
     }
 }
```

Reply:
I don't see any benefit to this. It just makes the code less readable?